### DISCUSSION TITLE
Convert chia-blockchain-gui to a client / server model

### DISCUSSION THREAD

**Discussion by oe3gwu** [Chia-Network/chia-blockchain-gui#2619] [Ideas]
You have the CLI client that can run as daemon. It would be nice if you rework the CHIA GUI so it can connect to those CLI Daemons. That way I can have 1 GUI for all my Chia Farmers that are reachable via VPN but not on the same site. This rework (or a sperate GUI "Thin" Client shouldn't be that hard to create, I assume.

**Comment by BrandtH22** [comment]
Hey @oe3gwu , thank you for the feature request. The team is working on a number of other enhancements and cannot provide any estimate as to when this will be reviewed for the potential of inclusion. 

In the meantime it might be worth looking into the official chia farming dashboard which many users have setup for multi farm environments to act as a read only UI for their various clients. 

Here is a youtube overview of setting up the dashboard on a single machine:
https://www.youtube.com/watch?v=7NNbmuwf99s

I like the text tutorial on chia decentral also:
https://chiadecentral.com/chia-farming-dashboard/

You can setup an instance of the chia exporter on each client then have those feed into a single grafana dashboard to view specs from all devices in a single location. 

This would not allow you to spend any funds or otherwise interact with the clients in a destructive manner so many users find this solution to be a more secure option than the development of a full client/server model that would expose more of those destructive actions.
